### PR TITLE
feat(): add implemented API feature list

### PR DESCRIPTION
**Pull request by joStrauss**
This list is based upon the git history of the framework and changelog of the osb-api

### CODE REVIEW

File: pages/osb-api-features.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Originating Identity: Often a service broker will need to know the identity of the user that initiated the request from the platform. For example, this might be needed for auditing or authorization purposes. In order to facilitate this, the platform will need to provide this identification information to the broker on each request. 
+
+- Service Metadata: a field to store metadata objects in service definitons and plans.
+
+## Open Service Broker API v2.14
+
+- fetching: Added GET endpoints for service instances and bindings.
+
+- Asynchronous Bindings: Bindings can be performed async, and a polling endpoint has been added.
```
Comment (chbrinker) [inline pages/osb-api-features.md]: async --> asynchronously

Comment (chbrinker) [reply pages/osb-api-features.md]: Either and or make two sentences. Remove the commata

File: pages/osb-api-features.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+- 
+
+
+## Open Service Broker API v2.15
+
+- Request Identity: A Platform might wish to uniquely identify a specific request as it flows throughout the system. For example, this might be used for logging for request tracing purposes.
```
Comment (chbrinker) [inline pages/osb-api-features.md]: Platform -> platform

Comment (chbrinker) [reply pages/osb-api-features.md]: " as it flows throughout the system" sounds odd

Comment (chbrinker) [reply pages/osb-api-features.md]: "for logging for request " thats no correct English. And I am even not sure what you want to explain here.

**Review by chbrinker — CHANGES_REQUESTED** [review]

File: pages/osb-api-features.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+
+### Implemented API Features
+
+This Documents sums up all OSB-API features, that got added after v2.12, the frameworks implements so far. The framework supports all API features up to version 2.14 and is currently upgraded to support 2.15. This is not a detailed documentation of these features, but a overview for developers. Please refer to OSB-API Docs for more information.
```
Comment (marberger) [inline pages/osb-api-features.md]: "is currently **being** upgraded to support"

File: pages/osb-api-features.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+- Originating Identity: Often a service broker will need to know the identity of the user that initiated the request from the platform. For example, this might be needed for auditing or authorization purposes. In order to facilitate this, the platform will need to provide this identification information to the broker on each request. 
+
+- Service Metadata: a field to store metadata objects in service definitons and plans.
+
+## Open Service Broker API v2.14
+
+- Fetching: Added GET endpoints for service instances and bindings.
+
+- Asynchronous Bindings: Bindings can be performed aynchronous and a polling endpoint has been added.
```
Comment (marberger) [inline pages/osb-api-features.md]: "aynchronous" -> missing "s"
Don't you use an adverb at this point? -> asynchronously

File: pages/osb-api-features.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (marberger) [inline pages/osb-api-features.md]: Missing Table of Contents with Links.

File: pages/osb-api-features.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- Fetching: Added GET endpoints for service instances and bindings.
+
+- Asynchronous Bindings: Bindings can be performed aynchronous and a polling endpoint has been added.
+
+## Open Service Broker API v2.15
+
+- Request Identity: A platform might wish to uniquely identify a specific request. This might be used for logging and request tracing purposes.
+
+- Maintenance Info: The service catalogs now holds an object to specify consequences of an provision or update action. This gives the user information about the steps initiated by the service broker when triggering such an action.
```
Comment (marberger) [inline pages/osb-api-features.md:41]: Missing footer with link to next and previous page.

**Review by marberger — CHANGES_REQUESTED** [review]
The osb-api-features site is missing the formal parts like the table of contents and footer links. 

Furthermore no links to the new page have been added to any other page (table of contents) and no footer links to the previous and next page for already existing pages.

File: pages/ide-runtime.md [hunk 7]
```diff
@@ -5,18 +5,18 @@
 3. [Configuring a Service Broker](configure-service-broker.md)
 4. [Service Keys](service-keys.md)
 5. [Backup Agent](backup-agent.md)
-6. [Development](development.md)
-7. [IDE & Runtime](ide-runtime.md)
-8. [Contribution](contribution.md)
-9. [License](license.md)
+7. [Open Service Broker API Features](osb-features.md)
```
Comment (marberger) [inline pages/ide-runtime.md:9]: 6. Development is missing

File: pages/license.md [hunk 8]
```diff
@@ -6,9 +6,10 @@
 4. [Service Keys](#service-keys)
 5. [Backup Agent](backup-agent.md)
 6. [Development](development.md)
-7. [IDE & Runtime](ide-runtime.md)
+7. [Open Service Broker API Features](osb-features.md)
+8. [IDE & Runtime](ide-runtime.md)
 8. [Contribution](contribution.md)
```
Comment (marberger) [inline pages/license.md]: No 9th item

File: pages/osb-api-features.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
+# Table of Contents
+
+1. [Open Service Broker Framework (by evoila)](../README.md)
+2. [Components](components.md)
+3. [Configuring a Service Broker](configure-service-broker.md)
+4. [Service Keys](service-keys.md)
+5. [Backup Agent](backup-agent.md)
+6. [Development](#development)
+7. [Open Service Broker API Features](osb-features.md)
+    * [7.1 Open Service Broker API v2.13](#open-service-broker-api-v2.13)
```
Comment (marberger) [inline pages/osb-api-features.md:10]: Three times 7.1

**Review by marberger — CHANGES_REQUESTED** [review]

**Review by marberger — APPROVED** [review]

**Pull request merged**